proxy.js

```diff
@@ -171,4 +171,4 @@
 
   scope.Proxy['revocable'] = scope.Proxy.revocable;
   scope['Proxy'] = scope.Proxy;
-})(typeof module !== 'undefined' && module['exports'] ? global : window);
+})(typeof process !== "undefined" && {}.toString.call(process) === "[object process]" ? global : typeof WorkerGlobalScope !== 'undefined' && self instanceof WorkerGlobalScope ? self : window);
```

Comment:
No, the point is not to shim Node.js APIs, just provide CommonJS support. Since the browser doesn't have a `global` property, using the module.exports technique will throw with loaders that support CommonJS.

I'll update it to your suggestion.  Wasn't aware that window.self was a thing now. Thanks!